### PR TITLE
Enforce multi part contributions restrictions

### PR DESCRIPTION
**Pull request by meiamsome**
Adds linting for #511

### CODE REVIEW

**Comment by meiamsome** [comment]
Whoops, accidentally tagged @blinkityblank for review - sorry!

File: unit_testing/yaml.test.js [hunk 1]
```diff
@@ -120,32 +122,41 @@ const checkFolder = (videoFormat, previousPath, folder) => describe(folder, () =
...
       // If we're in a standalone
       if(videoString === fileNumber) return;
       // If the video has parts, check we match the last one only.
       let parts = fileNumber.split('.');
       if(videoString === parts[parts.length - 1]) return;
       throw new Error('Expected file numbering to match internal numbering');
-    })
+    });
+
+    test('has no contributions if it\'s in a multi-part video but not the first part', () => {
```
Comment (meiamsome) [inline unit_testing/yaml.test.js]: If anyone can think of a better descriptor, that'd be great!

**Comment by meiamsome** [comment]
@blinkityblank Yep, that's done in #511! :D

**Comment by blinkityblank** [comment]
Awesome! And it's great you tagged me in this since I'm the one who usually creates those Coding Challenges files, it's good to know!

File: unit_testing/yaml.test.js (hunk 1 above)
Comment (nilsweb) [reply unit_testing/yaml.test.js]: Hmm, tough one... Maybe 'has valid contributions' but I can't think of anything better either.

Comment (KristianMischke) [reply unit_testing/yaml.test.js]: What about 'has no contributions for multi-part videos beyond part one'

**Pull request merged**